vault/changelog.md

```diff
@@ -2,10 +2,31 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1658856871637
+updated: 1659373582981
 created: 1601508213606
 nav_order: 2
 ---
+## 0.106.0
+
+### Breaking
+
+- the `Delete Node` command is now replaced with the `Delete` command (if you haven't set any custom keyboard shortcuts for `Delete Node`, no change is necessary)
+- the `Show Preview` command is now replaced with the `Toggle Preview` command (if you haven't set any custom keyboard shortcuts for `Show Preview`, no change is necessary)
+
+### Enhancements
+
+- enhance(edit): remove delete node command (#3285) @kevin
+- enhance(retrieve): remove `Show Preview` command (#3276) @kevin
+- enhance(workspace): `Fix It` button in warning toaster for duplicate note id (#3237) @joshi
+- enhance(retrieve): fix default keyboard shortcut of toggle preview on windows (#3276) @kevin
```

Comment:
`enhance(retrieve): fix default keyboard shortcut of toggle preview on windows (#3276) @kevin` 
can be updated to : 
`- fix(retrieve): windows keybindings for toggle preview(#3323) @kevin`